src/util.js

```diff
@@ -120,6 +120,10 @@ export function updateContainers() {
         varlinkCall(PODMAN, "io.podman.ListContainers")
                 .then(reply => {
                     let newContainersMeta = reply.containers;
+                    if (newContainersMeta === null) {
+                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                        return;
+                    }
```

Comment:
Out of interest, is that related to this PR? It seems odd that ListContainers would ever answer with a null value, instead of an empty list. It's okay to handle it like this, although it would be simpler to provide a fallback value:

    let newContainersMeta = reply.containers || [ ];